$NetBSD: patch-ab,v 1.1 2005/11/03 01:47:08 rillig Exp $

Don't use the *f() math functions on Solaris. The Solaris preprocessor
does not define the symbol "__sun__", so we have to check for "__sun",
too. The latter is mentioned in the cpp(1) man page.

--- src/glitzint.h.orig	2005-07-04 12:33:01.000000000 +0200
+++ src/glitzint.h	2005-11-03 02:12:47.808896500 +0100
@@ -33,7 +33,7 @@
 
 #include "glitz.h"
 
-#if defined(__APPLE__) || defined(__sun__)
+#if defined(__APPLE__) || defined(__sun__) || defined(__sun)
 # define floorf(a)    floor (a)
 # define ceilf(a)     ceil (a)
 # define sinf(a)      sin (a)
